internal_transfers/actions/tests/e2e/getInternalImbalance.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { getInternalImbalance, MinimalTxData } from "../../src/accounting";
+import { TenderlySimulator } from "../../src/simulate/tenderly";
+import { ethers } from "ethers";
+
+const simulator = new TenderlySimulator(
+  process.env["TENDERLY_USER"] || "INVALID_USER",
+  process.env["TENDERLY_PROJECT"] || "TENDERLY_PROJECT",
+  process.env["TENDERLY_ACCESS_KEY"] || "TENDERLY_ACCESS_KEY"
+);
+
+async function TxDataFromHash(txHash: string): Promise<MinimalTxData> {
+  const provider = ethers.getDefaultProvider(
+    process.env["NODE_URL"] || "NODE_URL"
+  );
+  const transaction = await provider.getTransactionReceipt(txHash);
+  if (transaction === null) {
+    throw new Error(`invalid transaction hash ${txHash} - try again`);
+  }
+  const { from, hash, logs } = transaction;
+  const tenderlyLogs = logs.map((log) => {
+    const { address, data, topics } = log;
+    return {
+      address,
+      data,
+      // had to Map it to make a copy of a readonly field.
+      topics: topics.map((value) => value),
+    };
+  });
+
+  return {
+    from,
+    hash,
+    logs: tenderlyLogs,
+  };
+}
+describe.skip("getInternalImbalance(transaction, simulator)", () => {
+  test.skip("throws when no competition found", async () => {
+    const txHash =
+      "0x08100e7ba81be84ee0bdce43db6640e2f992ec9991a740a689e97d20dea9dafa";
+    const transaction = await TxDataFromHash(txHash);
+    await expect(getInternalImbalance(transaction, simulator)).rejects.toThrow(
+      `No competition found for ${txHash}`
+    );
+  });
```

Comment:
This test appears to be redundant (same name `throws when no competition found`).

Reply:
Could it be that its just the name which is redundant? Or do I actually have the same test twice?

Reply:
It looks the same to me with the exception that the simulator is a fake one and not a real one ([here](https://github.com/cowprotocol/solver-rewards/blob/8758e930a89dc4ded8a98c11718c382da16069ba/internal_transfers/actions/tests/accounting.spec.ts#L5-L22)). Since the simulation isn't important for the test I'd remove one (it makes more sense to keep one that isn't ignored by default) but if you believe the simulator can become important you can also keep this.

Reply:
Ok, right so one is for fake simulator and the other for a real one. I think the point to be made here is do we really expect the same kind of behaviour in these two different scenarios? Wouldn't we prefer to error if we did not supply the correct credentials?

Reply:
Sounds good to me, I don't think it should block this PR.